site/_posts/2022-04-15-announcing-fusionauth-1.36.md

```diff
@@ -0,0 +1,74 @@
+
+<!--more-->
+
+These releases contained features, enhancements, and bug fixes. Please see the [release notes](/docs/v1/tech/release-notes#version-1-36-1) for a full breakdown of the changes between 1.35 and 1.36. 
+
+There are a few improvements that I wanted to call out specifically.
+
+## SCIM 2 support
+
+[SCIM 2](http://www.simplecloud.info) (System for Cross-domain Identity Management), a standardized API for identity management sponsored by the IETF, allows user records to be read, written, and shared via HTTP using a simple JSON schema. In a [long-awaited feature](https://github.com/FusionAuth/fusionauth-issues/issues/106), FusionAuth now supports SCIM formats and operations as well as our own API.
+
+This feature is particularly helpful in cases where you want to use a third-party SCIM-compliant backend to provision users and keep user data in sync. SCIM helps you manage users across systems like GitHub, Salesforce, and Trello, among others.
+
+SCIM data formats vary wildly so new lambdas are available. The lambdas will take care of mapping from the SCIM provided user data into the FusionAuth user data schema and vice versa. This allows for a large degree of integration flexibility.
+
+This functionality is only available for our Enterprise users; learn more by visiting [the pricing page](/pricing) or [contacting our sales team](/contact).
+
+{% include _image.liquid src="/assets/img/blogs/release-1-36/scim-tenant-settings.png" alt="SCIM tenant settings" class="img-fluid" figure=true %}
+
+## Nintendo Online Identity Provider
+
+Nintendo Online has been added as an identity provider, allowing users to log in using their Nintendo Online accounts.
+
+{% include _image.liquid src="/assets/img/blogs/release-1-36/nintendo-login.png" alt="Nintendo added as Idp." class="img-fluid" figure=true %}
+
+This feature is available to our Enterprise and Essentials license holders as part of the “Advanced and gaming Idp” suite. Learn more about these editions by visiting [the pricing page](/pricing) or [contacting our sales team](/contact).
+ 
+## Webhook for user identity provider linking events
+
+A new webhook event is available for developers to trigger an action when a user links or unlinks an identity provider such as Google or Facebook.
+
+This will allow developers to write a subsystem that immediately notices any new IdP accounts their users associate with their FusionAuth account (e.g. Google, Facebook, LinkedIn) and when these accounts are unlinked from FusionAuth. Another system listens for those events and handles them: for instance it can record the data for analytics, or adjust functionality which is only available if you have an account at a certain IdP, like a special in-game item is only available to Xbox users.
+
+A typical user scenario might go as follow:
+
+* A user, Alice, signs in to GameA using 'login with XBox' and FusionAuth. A link event is sent.
```

Comment:
What's GameA (I was using that as an example)? I don't know any popular games, but maybe you could go with `ExampleGame` or `AcmeGame` or 'some game'?